sdk/schemaregistry/azure-data-schemaregistry/src/test/java/com/azure/data/schemaregistry/SchemaRegistryAsyncClientCustomTests.java

```diff
@@ -68,10 +65,8 @@ protected void beforeTest() {
 
         if (interceptorManager.isPlaybackMode()) {
             builder.httpClient(buildAsyncAssertingClient(interceptorManager.getPlaybackClient()));
```

Comment:
unrelated but we should be using/validating with the asserting client always and not just int playback mode only.

Reply:
The uses of `.httpClient` all have the assertingClient wrapped around the client.